### PR TITLE
Making UI changes on splash,home, charts and setting screens. Routing

### PR DESCRIPTION
**Pull request by Francmape**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.3"
+        classpath 'com.android.tools.build:gradle:7.1.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31"
```
Comment (Wamae) [inline build.gradle:14]: `classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10"`
Bump to the latest version.

File: build.gradle [hunk 2]
```diff
@@ -10,7 +10,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.3"
```
Comment (Wamae) [inline build.gradle]: Bump this to the latest version, we want to be working with the latest and greatest.

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
```
Comment (Wamae) [inline .idea/misc.xml]: This file should not be saved in the repo. Gitignore will help.

**Review by Wamae — CHANGES_REQUESTED** [review]

**Comment by Francmape** [comment]
Changes effected, kindly confirm. @Wamae

**Pull request merged**